Sources/Basic/TerminalController.swift

```diff
@@ -78,11 +78,13 @@ public final class TerminalController {
             return width
         }
 
+#if TIOCGWINSZ
```

Comment:
Is TIOCGWINSZ defined on powerpc64le? Maybe it is a macro which Swift can't import?